test/known_issues/test-vm-proxy-failure-CP.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+require('../common');
+
+const vm = require('vm');
+
+// Sandbox throws while copying properties despite no code being run
+const handler = {
+    getOwnPropertyDescriptor: (target, prop) => {
+      throw new Error('whoops');
+    }
+};
+const sandbox = new Proxy({foo: 'bar'}, handler);
+const context = vm.createContext(sandbox);
+
+vm.runInContext('', context);
```

Comment:
Can you add an assert? I'd be nice to move this file from `known_issue` to `test` once the issue is addressed, and a test without an assert would be confusing.